### PR TITLE
OEP-54 - Core Contributors - Define work scope

### PR DESCRIPTION
**Pull request by antoviaque**
This update to OEP-54 adds a link to the reference definition of what is being considered "core contributor work", which will live in the wiki. 

The corresponding review can be found in the forum at https://discuss.openedx.org/t/final-review-core-contributors-work-scope/8821

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @antoviaque!  Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.

**Comment by itsjeyd** [comment]
Hey @mphilbrick211, could you please update the status of this PR on the contributions board to *In Review*? Even though the review is happening via the [forums](https://discuss.openedx.org/t/final-review-core-contributors-work-scope/8821) instead of here, it seems like that would be the appropriate status.

**Comment by mphilbrick211** [comment]
All set @itsjeyd!

**Comment by itsjeyd** [comment]
Hey @antoviaque, now that the [review period](https://discuss.openedx.org/t/core-contributors-work-scope-definition-final-review/8821/3?u=itsjeyd) has ended, how should we proceed with this PR? Will it simply need a 👍 on the diff or are there other things to sort out first?

CC @mphilbrick211

**Comment by antoviaque** [comment]
@itsjeyd Thanks for checking on it! I now need to do a proper recap & response to the review thread, and specify a conclusion & next steps. Depending on the response to it, it might then either be adopted, or another round of review started. I'll be sure to post a corresponding update here.

**Comment by itsjeyd** [comment]
Sounds good, thanks @antoviaque.

**Comment by antoviaque** [comment]
@itsjeyd Done! I've posted in the [forum thread](https://discuss.openedx.org/t/core-contributors-work-scope-definition-final-review/8821/4), and updated the [wiki page linked from the current PR diff](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3593502844/Core+Contributor+-+Work+Scope).

I believe we should be good to merge this PR?

**Comment by itsjeyd** [comment]
@antoviaque Yep, looks like it just needs formal approval from a reviewer. I was going to ping OEP-54's arbiter, but since that's you, I think we'll need to find someone else 😅 

@jmbowman Since arch-bom owns this repo, would you be able to review here (or let us know who to ask instead)?

File: oeps/processes/oep-0054-core-contributors.rst
Comment (jmbowman) [inline oeps/processes/oep-0054-core-contributors.rst:120]: The link syntax is malformed, I think you're missing the initial backtick.  It should render correctly in GitHub via "View File" from the PR once fixed, or you can check the Sphinx rendering locally.  But word of warning, I just wrote a small essay on how to do that and what it says about the current state of our developer experience: https://docs.google.com/document/d/1tM6JLczGRTnvaABhWclAPe22urzYdqG39PQk-tixIv8/edit#

**Review by jmbowman — COMMENTED** [review]
Other than the link formatting, looks good to me.

**Comment by itsjeyd** [comment]
Hey @antoviaque, it looks like this is very close to merging now! But let me know if there's anything left here that I could help with.

File: oeps/processes/oep-0054-core-contributors.rst
Comment (antoviaque) [inline oeps/processes/oep-0054-core-contributors.rst:120]:
```suggestion
`<https://openedx.atlassian.net/wiki/spaces/COMM/pages/3593502844/Core+Contributor+-+Work+Scope>`_
```

Comment (antoviaque) [reply oeps/processes/oep-0054-core-contributors.rst:120]: @sarina It looks like I can't modify the branch I had originally created here - and even as a suggestion like here github doesn't let me apply the change. I assume this is due to the permissions changes I've had lately?

Would it be possible to have r/w access on this repo? Besides allowing me to fix this, I would also be able to merge PRs like on this OEP.

**Comment by antoviaque** [comment]
@itsjeyd Thanks for the ping! That helps me to notice the activity on the PR :)

File: oeps/processes/oep-0054-core-contributors.rst
Comment (antoviaque) [reply oeps/processes/oep-0054-core-contributors.rst:120]: @jmbowman Thank you for the review - and sorry that it caused you such a hassle to review. :/ Even without running the whole build like you did, I could have caught it in the github render. And now I know to look for the rendered version in the tests - here https://open-edx-proposals--417.org.readthedocs.build/en/417/processes/oep-0054-core-contributors.html#core-contributor-program-administration

I'm currently blocked on not being able to update the branch from my PR, but I'll make sure to fix this once it's solved.

Comment (antoviaque) [reply oeps/processes/oep-0054-core-contributors.rst]: @jmbowman The typo show now be fixed - you can see the current output at https://open-edx-proposals--417.org.readthedocs.build/en/417/processes/oep-0054-core-contributors.html#core-contributor-program-administration

**Review by jmbowman — APPROVED** [review]

**Comment by antoviaque** [comment]
@jmbowman Thanks for the review and approval! I'll merge this now then.

**Comment by openedx-webhooks** [comment]
@antoviaque 🎉 Your pull request was merged!  Please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/open-edx-proposals&entry.867055334=https://github.com/openedx/open-edx-proposals/pull/417&entry.1484655318=https://github.com/antoviaque&entry.752974735=2022-11-30+20:57&entry.1917517419=2023-02-20+09:22&entry.2133058324=Yes) so we can improve your experience in the future.

**Pull request merged**
